app/models/registration/pages/CharityOrTrust.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.registration.pages
+
+import models.{Enumerable, WithName}
+import play.api.libs.json.Json
+import viewmodels.RadioOption
+
+sealed trait CharityOrTrust
+
+object CharityOrTrust extends Enumerable.Implicits {
+
+  case object Charity extends WithName("Charity") with CharityOrTrust
```

Comment:
@JohnWhitfield should these be capital letters? How do you normally structure ids for UI tests?